.github/workflows/create-summary-pr.yml

```diff
@@ -0,0 +1,24 @@
+name: Create PR for load test results
```

Comment:
Shall we remove this file and add the https://github.com/ballerina-platform/module-ballerina-grpc/blob/master/.github/workflows/process-load-test-result.yml file. This workflow still does not complete since the `ballerina_reviewer_bot_token` is not yet added. But it works upto the PR creation step. We can approve and merge it manually for now.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-io/pull/416/commits/e6fbfa5455c5f9f8bd09ee44ad29cc21fca7edd4